helm/vitess/templates/_orchestrator-conf.tpl

```diff
@@ -0,0 +1,163 @@
+###################################
+# Orchestrator Config
+###################################
+{{- define "orchestrator-config" -}}
+# set tuple values to more recognizable variables
+{{- $orc := index . 0 -}}
+{{- $namespace := index . 1 -}}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: orchestrator-cm
+data: 
+  orchestrator.conf.json: |-
+    {
+    "AccessTokenExpiryMinutes": 1440,
+    "AccessTokenUseExpirySeconds": 60,
+    "ActiveNodeExpireSeconds": 5,
+    "AgentPollMinutes": 60,
+    "AgentsServerPort": ":3001",
+    "AgentSSLCAFile": "",
+    "AgentSSLCertFile": "",
+    "AgentSSLPrivateKeyFile": "",
+    "AgentSSLSkipVerify": false,
+    "AgentSSLValidOUs": [
+    ],
+    "AgentsUseMutualTLS": false,
+    "AgentsUseSSL": false,
+    "ApplyMySQLPromotionAfterMasterFailover": true,
+    "AuditLogFile": "/tmp/orchestrator-audit.log",
+    "AuditToSyslog": false,
+    "AuthenticationMethod": "",
+    "AuthUserHeader": "",
+    "AutoPseudoGTID": false,
+    "BackendDB": "sqlite",
+    "BinlogEventsChunkSize": 10000,
+    "CandidateInstanceExpireMinutes": 60,
+    "ClusterNameToAlias": {
+        "127.0.0.1": "test suite"
+    },
+    "CoMasterRecoveryMustPromoteOtherCoMaster": true,
+    "DataCenterPattern": "[.]([^.]+)[.][^.]+[.]mydomain[.]com",
+    "Debug": true,
+    "DefaultInstancePort": 3306,
+    "DefaultRaftPort": 10008,
+    "DetachLostSlavesAfterMasterFailover": true,
+    "DetectClusterAliasQuery": "SELECT value FROM _vt.local_metadata WHERE name='ClusterAlias'",
+    "DetectClusterDomainQuery": "",
+    "DetectInstanceAliasQuery": "SELECT value FROM _vt.local_metadata WHERE name='Alias'",
+    "DetectPromotionRuleQuery": "SELECT value FROM _vt.local_metadata WHERE name='PromotionRule'",
+    "DetectPseudoGTIDQuery": "",
+    "DetectSemiSyncEnforcedQuery": "SELECT @@global.rpl_semi_sync_master_wait_no_slave AND @@global.rpl_semi_sync_master_timeout > 1000000",
+    "DiscoverByShowSlaveHosts": true,
+    "EnableSyslog": false,
+    "ExpiryHostnameResolvesMinutes": 60,
+    "FailureDetectionPeriodBlockMinutes": 60,
+    "GraphiteAddr": "",
+    "GraphiteConvertHostnameDotsToUnderscores": true,
+    "GraphitePath": "",
+    "HostnameResolveMethod": "none",
+    "HTTPAuthPassword": "",
+    "HTTPAuthUser": "",
+    "InstanceBulkOperationsWaitTimeoutSeconds": 10,
+    "InstancePollSeconds": 12,
+    "ListenAddress": ":3000",
+    "MasterFailoverLostInstancesDowntimeMinutes": 0,
+    "MySQLConnectTimeoutSeconds": 1,
+    "MySQLHostnameResolveMethod": "none",
+    "MySQLTopologyCredentialsConfigFile": "",
+    "MySQLTopologyMaxPoolConnections": 3,
+    "MySQLTopologyPassword": "orc_client_user_password",
```

Comment:
yay for plaintext passwords, I guess. Or use `MySQLTopologyCredentialsConfigFile`

Reply:
@enisoc Right now that password is hardcoded into `init_db.sql`. Do you have any concerns about the plaintext password? Since nothing else is locked down by default, it seems ok to me.

Reply:
Eventually, I want to generate random passwords and self-signed certs and distribute them via the Vitess Operator. For now, I wouldn't block this PR on it since everything else in our example is using insecure defaults.

Reply:
👍